### DISCUSSION TITLE
Add option for no welcome intro animation

### DISCUSSION THREAD

**Discussion by jai-x** [ppy/osu#15261] [Ideas] [answered]
Currently, you either select a specific intro animation (welcome, circles, triangles) or a random one in the settings.
Would be nice if there was an option for no intro animation such that the flow becomes (Game boot -> Disclaimer -> Menu).

**Comment by peppy** [comment, marked as answer]
I don't see this happening ever. The welcome is part of what osu! is to me.

What's the use case?

**Reply by jai-x** [reply]
No real use case other than personal preference. I can understand wanting to keep it as part of the vision of the game.

**Comment by PercyDan54** [comment]
I prefer to skip the disclaimer instead

**Reply by peppy** [reply]
yes well that won't be here forever. just deal with it for now. it also helps hide the load time of the game.
